src/components/table/draggable-table-cell/draggable-table-cell.style.js

```diff
@@ -0,0 +1,27 @@
+import styled, { css } from 'styled-components';
+import StyledTableCell from '../table-cell/table-cell.style';
+import baseTheme from '../../../style/themes/base';
+
+const StyledDraggableTableCell = styled(StyledTableCell)`
+  ${applyClassicDraggableStyling}
+`;
+
+function applyClassicDraggableStyling() {
+  return css`
+    padding: 0;
+    width: 21px;  
+    user-select: none;
+
+    ${StyledTableCell}:first-child {
+      padding-left: 0;
+    }
+  `;
+}
```

Comment:
is that method needed? You could just render this style directly in `StyledDraggableTableCell `

Reply:
Yeah I've removed it, I had it that way before I realised there was no difference between the classic and modern theme styles